renting_platform.sql

```diff
@@ -0,0 +1,159 @@
+-- Created by Vertabelo (http://vertabelo.com)
+-- Last modification date: 2022-09-12 14:31:40.494
+
+-- tables
+-- Table: adress
+CREATE TABLE adress (
+    id int NOT NULL AUTO_INCREMENT,
+    street varchar(45) NOT NULL,
+    house_number int NULL,
+    city_id int NOT NULL,
+    CONSTRAINT adress_pk PRIMARY KEY (id)
+);
+
+-- Table: cash_withdrawal_from_lessee_to_service
+CREATE TABLE cash_withdrawal_from_lessee_to_service (
+    id int NOT NULL,
+    price int NOT NULL,
+    reservation_id int NOT NULL,
+    CONSTRAINT cash_withdrawal_from_lessee_to_service_pk PRIMARY KEY (id)
+);
+
+-- Table: cash_withdrawal_from_service_to_owner
+CREATE TABLE cash_withdrawal_from_service_to_owner (
+    id int NOT NULL,
+    price int NOT NULL,
+    cash_withdrawal_from_lessee_to_service_id int NOT NULL,
+    CONSTRAINT cash_withdrawal_from_service_to_owner_pk PRIMARY KEY (id)
+);
+
+-- Table: city
+CREATE TABLE city (
+    id int NOT NULL AUTO_INCREMENT,
+    name varchar(35) NOT NULL,
+    region_id int NOT NULL,
+    CONSTRAINT city_pk PRIMARY KEY (id)
+);
+
+-- Table: dwelling
+CREATE TABLE dwelling (
+    id int NOT NULL AUTO_INCREMENT,
+    area int NOT NULL,
+    floor int NOT NULL,
+    rooms_number int NOT NULL,
+    adress_id int NOT NULL,
+    dwelling_owner_id int NOT NULL,
+    reservation_id int NOT NULL,
+    description varchar(100) NOT NULL,
+    CONSTRAINT dwelling_pk PRIMARY KEY (id)
+);
+
+-- Table: dwelling_owner
+CREATE TABLE dwelling_owner (
+    id int NOT NULL AUTO_INCREMENT,
+    name varchar(25) NOT NULL,
+    surname varchar(50) NOT NULL,
+    phome varchar(12) NOT NULL,
+    email varchar(35) NOT NULL,
+    CONSTRAINT dwelling_owner_pk PRIMARY KEY (id)
+);
+
+-- Table: dwelling_photo
+CREATE TABLE dwelling_photo (
+    id int NOT NULL,
+    dwelling_id int NULL,
+    name varchar(35) NULL,
+    CONSTRAINT dwelling_photo_pk PRIMARY KEY (id)
+);
+
+-- Table: lessee
+CREATE TABLE lessee (
+    id int NOT NULL AUTO_INCREMENT,
+    name varchar(25) NOT NULL,
+    surname varchar(50) NOT NULL,
+    phone varchar(12) NOT NULL,
+    email varchar(35) NOT NULL,
+    CONSTRAINT lessee_pk PRIMARY KEY (id)
+);
+
+-- Table: lessee_feedback
+CREATE TABLE lessee_feedback (
+    rating int NOT NULL AUTO_INCREMENT,
+    response varchar(100) NOT NULL,
+    lessee_id int NULL,
+    CONSTRAINT lessee_feedback_pk PRIMARY KEY (rating)
```

Comment:
I guess rating was a bad choice of PK for this table.
AUTO_INCREMENT means the first record in this table will have rating=1, the second record will have rating=2, the third will have rating=3 and so on.
But even if you didn't mean it to AUTO_INCREMENT, even if you wanted just to store the rating provided by other users, tell me: what happens if two lessees both have rating=99?

Reply:
Yep, you are absolutely right, I will get an error when both of them have rating(99) because in my scheme rating was PK(It cant be duplicate).
AUTO_INCREMENT is the same one(wrong choice).
Thanks for reviewing.